site/src/ThemeSelector.css

```diff
@@ -33,4 +33,9 @@ svg.icon-layers > path.primary {
 
 svg.icon-layers > path.secondary {
   fill: cornflowerblue;
-}
\ No newline at end of file
+}
+
+svg.icon-layers-dark {
```

Comment:
only substantive change, adding a adaptive background coloring and border radius to match the light mode shape. All other changes in file are formatting.